flutter-idea/src/io/flutter/FlutterInitializer.java

```diff
@@ -130,6 +134,23 @@ public void moduleAdded(@NotNull Project project, @NotNull Module module) {
     }
 
     if (hasFlutterModule) {
+      if (PluginManagerCore.getPlugin(PluginId.getId("org.jetbrains.android")) != null) {
+        if (!FlutterModuleUtils.hasAndroidModule(project)) {
```

Comment:
do you need these two conditions to be on separate lines?

Reply:
I changed the nested if-stmts to a single if-stmt, but the conditions are still on separate lines.